### PR TITLE
Add End-to-end encryption support for triggerbatch + tests

### PR DESCRIPTION
**Pull request by damdo**
- It adds expected End-to-end encryption support for `triggerbatch`
- It adds more tests for corner cases in `triggerbatch`

### CODE REVIEW

File: src/Pusher.php [hunk 1]
```diff
@@ -554,24 +554,31 @@ public function trigger($channels, $event, $data, $socket_id = null, $debug = fa
      */
     public function triggerBatch($batch = array(), $debug = false, $already_encoded = false)
     {
-        $query_params = array();
+        $has_encrypted_channel = false;
```
Comment (WillSewell) [inline src/Pusher.php]: This is unused.

File: tests/acceptance/triggerBatchTest.php [hunk 2]
```diff
@@ -33,15 +33,164 @@ public function testTLSPush()
...
+        $pc->setLogger(new TestLogger());
+
+        $batch = array();
+        $batch[] = array('channel' => 'test_channel', 'name' => 'my_event', 'data' => array('my' => 'data'));
+        $batch[] = array('channel' => 'mio_canale', 'name' => 'my_event2', 'data' => array('my' => 'data2'));
+        $string_trigger = $pc->triggerBatch($batch);
+        $this->assertTrue($string_trigger, 'Failed to triggerBatch Multiple Events');
+    }
+
+    public function testTriggerBatchWithSecretString()
```
Comment (WillSewell) [inline tests/acceptance/triggerBatchTest.php]: Why "Secret" string?

File: src/Pusher.php (hunk 1 above)
Comment (damdo) [reply src/Pusher.php]: Right.

File: tests/acceptance/triggerBatchTest.php (hunk 2 above)
Comment (damdo) [reply tests/acceptance/triggerBatchTest.php]: Yeah, I will definitely come up with a better name :)
